src/Compilers/CSharp/Portable/Declarations/DeclarationTreeBuilder.cs

```diff
@@ -975,12 +1091,11 @@ private static bool CheckMemberForAttributes(Syntax.InternalSyntax.CSharpSyntaxN
         }
 
         private static void AddNonTypeMemberNames(
-            Syntax.InternalSyntax.CSharpSyntaxNode member, ImmutableSegmentedHashSet<string>.Builder set, ref bool anyNonTypeMembers, bool skipGlobalStatements)
+            Syntax.InternalSyntax.CSharpSyntaxNode member, HashSet<string> set)
```

Comment:
this was broken into two helpers as this helper will not be called in the case where we can find the cached names.